tools/restatectl/src/commands/replicated_loglet/digest.rs

```diff
@@ -120,19 +160,19 @@ async fn get_digest(connection: &ConnectionInfo, opts: &DigestOpts) -> anyhow::R
     heading.extend(vec![Cell::new("ISSUES")]);
     records_table.set_header(heading);
 
+    let mut checker = NodeSetChecker::new(&nodeset, &nodes_config, &loglet_ref.params.replication);
     for (offset, responses) in digests.iter() {
+        checker.fill_with_default();
         if *offset >= digests.max_local_tail() {
             break;
         }
         if *offset == known_global_tail.latest_offset() {
             // divider to indicate that everything after global tail
             records_table.add_row(std::iter::repeat("────").take(nodeset.len() + 2));
```

Comment:
When trying the command out I got this output
Which conflicts with the comment here, the line should be after the value of 75095 not before it

![image](https://github.com/user-attachments/assets/7c274c07-cf96-486c-9d15-cb35e0b789c1)